cns/service/main.go

```diff
@@ -535,6 +535,14 @@ func main() {
 		}
 	}
 
+	go func() {
```

Comment:
Can we initialize this as how IpamPoolMonitor is started?

Reply:
Also can you check with Ashvin, do we need to add this thread for all scenarios? (Both Managed DNC and non-Managed as well?)

Reply:
I think Ashvin mentioned during syncing up meeting that update NC's host version(the version programmed in NMAgent) is necessary for all scenarios. But anyway, I can double check with him.

Reply:
yes, but you need to check if the NMA on this node supports the API without the token. If that is supported, you would call that API else call the API with the token. I don;t see that code change in this PR. GetNetworkContainerInfoFromHostWithoutToken is missing the correct implementation to call that NMA Api. nmagentclient.go has GetNetworkContainerVersion which is calling the api with tokens.

Reply:
The initialization is the same as IpamPoolMonitor except IpamPoolMonitor use context for context.done feature. Context.Done give us visibility(logging) when IpamPoolMonitor go routine received cancellation signal. Do we want this visibility for SyncHostNCVersion? If yes, I can add context feature to it.